File: JAVA/Garage.java
package  Java;

import java.util.ArrayList;
import java.util.List;

public class Garage {
    private String name;
    private double area;
    private List<Vehicle> vehicles;

    public Garage(String name, double area) {
        this.name = name;
        this.area = area;
        this.vehicles = new ArrayList<>();
    }

    public void addVehicle(Vehicle vehicle) {
        vehicles.add(vehicle);
    }

    public void displayVehicles() {
        System.out.println("Garage: " + name + ", Area: " + area + " sqm");
        System.out.println("Vehicles:");
        for (Vehicle vehicle : vehicles) {
            vehicle.displayInfo();
            System.out.println();
        }
    }
}

File: PHP/Garage.php
<?php
require_once 'Vehicle.php';

class Garage {
    private $name;
    private $area;
    private $vehicles = [];

    public function __construct($name, $area) {
        $this->name = $name;
        $this->area = $area;
    }

    public function addVehicle($vehicle) {
        $this->vehicles[] = $vehicle;
    }

    public function displayVehicles() {
        echo "Garage: " . $this->name . ", Area: " . $this->area . " sqm" . PHP_EOL;
        echo "Vehicles:" . PHP_EOL;
        foreach ($this->vehicles as $vehicle) {
            $vehicle->displayInfo();
            echo PHP_EOL;
        }
    }
}
?>
